QgisModelBaker/tests/test_export.py

```diff
@@ -393,6 +403,7 @@ def test_ili2db3_export_mssql(self):
             dataImporter.tool, 'ilimodels/CIAF_LADM')
         dataImporter.configuration.ilimodels = 'CIAF_LADM'
         dataImporter.configuration.dbschema = importer.configuration.dbschema
+        dataImporter.configuration.create_basket_col = False
```

Comment:
Why not just leaving it `False` by default? :) 

Just kidding, I guess your users need it active by default.

Reply:
hehe